pkg/storage/blockdev.go

```diff
@@ -190,3 +198,27 @@ func FilterEFISystemPartitions(devices []BlockDev) ([]BlockDev, error) {
 	}
 	return esps, nil
 }
+
+// GetMountpointByDevice gets the mountpoint by given
+// device name. Returns on first match
+func GetMountpointByDevice(devicePath string) (string, error) {
```

Comment:
I am not convinced by this interface: if the device doesn't have a mount point, it will return the empty string, and no error. The caller could check for the empty string, but this is inconsistent.

Is the lack of a mount point to be considered an error? And if not, shall we rather return a `nil` string pointer and a `nil` error to say "no error, but no mountpoint either"?

Reply:
Yeah let's use string pointer here